sqlest/src/main/scala/sqlest/extractor/ColumnExtractorSetters.scala

```diff
@@ -64,6 +65,14 @@ trait ColumnExtractorSetters {
             case (extractor, value) => extractor.settersFor(value)
           }.toList
 
+        case choiceExtractor: ChoiceExtractor[ResultSet, a, b] =>
+          choiceExtractor.extractors.map {
+            case extractor: Extractor[ResultSet, c] =>
+              Try { extractor.settersFor(value.asInstanceOf[c]) }.toOption
```

Comment:
This is basically a hack - is there a way we can use TypeTags / typeOf or something like that to avoid this?